### DISCUSSION TITLE
when will actions/cache be available on github enterprise?

### DISCUSSION THREAD

**Discussion by sw-fox** [actions/cache#658] [Q&A]
we are at the momenet migrating our ci/cd infrastructure to github. But without caching our pipelines need 3-4 times as long compared to our old buildpipeline with caching (GitlabCI)
This is a big stopper for us.

**Comment by N-Usha** [comment]
Thanks for reaching out @sw-fox . We are actively tracking the effort to support Cache on GHES and it will be tentatively delivered in a few quarters from now.  Hope that helps.

**Reply by sw-fox** [reply]
this means caching will be available in Q2/Q3 2022?
